tdms/include/interpolation_methods.h

```diff
@@ -105,7 +103,11 @@ class interpScheme {
 
         /* END FETCH METHODS */
         
-        // stores the index-offset that we require when extracting data from the field component arrays
+        /* Stores the index-offset that we require when extracting data from the field component arrays
+        Let F be the (split field) Yee cell components of some field component, and suppose we are interested in interpolating this field to the centre of the cell with index u.
+        Then the data point F[u - (index+1)] is plays the role of v[0] in the interpolation scheme.
```

Comment:
Is v[0] the same as v0 in the above ASCII diagrams, which explain interp1, interp2, etc. In particular:

 * v0    v1    v2    v3
 * o     o  x   o    o

Reply:
Yes - don't know why I included the square bracers around `[0]` here!